lib/session-file-helpers.js

```diff
@@ -135,8 +139,17 @@ var helpers = {
     });
 
     operation.attempt(function () {
-      fs.readJson(sessionPath, 'utf8', function (err, json) {
-        if (!err) return callback(null, helpers.isExpired(json, options) ? null : json);
+      fs.readFile(sessionPath, 'utf8', function (err, data) {
+
+        if (!err) {
+          var json
```

Comment:
replace with 

    var json = JSON.parse(options.encrypt ? this.decrypt(data, sessionId) : data);